sql/moz-fx-data-shared-prod/bigeye_derived/dashboard_service_v1/query.py

```diff
@@ -0,0 +1,169 @@
+"""Pull from BigEye API from dashboardService and upload to Big Query."""
+
+import json
+import os
+import time
+from typing import Any, Dict
+
+# Libraries specific for pulling from API and uploading to bigquery.
+import pandas as pd
+import requests
+from google.api_core.exceptions import GoogleAPIError
+from google.cloud import bigquery
+from requests.exceptions import RequestException
+
+# Configuration
+TARGET_PROJECT = os.getenv("TARGET_PROJECT", "moz-fx-data-shared-prod")
+TARGET_TABLE = os.getenv(
+    "TARGET_TABLE", "moz-fx-data-shared-prod.bigeye_derived.dashboard_service"
+)
+BIGEYE_API_KEY = os.environ["BIGEYE_API_KEY"]
```

Comment:
This needs to be set in the metadata.yaml file. Here's an example of this:
https://github.com/mozilla/private-bigquery-etl/blob/main/sql/moz-fx-data-shared-prod/cinder_external/jobs_raw_v1/metadata.yaml#L15-L17

Basically under scheduling we should have something like this:
```
scheduling:
  dag_name: bqetl_bigeye_derived
  secrets:
  - deploy_target: BIGEYE_API_KEY
    key: bqetl_bigeye_api_key # need to check if this secret is already defined in GSM, if not we need to create it.
```